examples/container_registries.tfvars

```diff
@@ -0,0 +1,56 @@
+container_registries = {
+  acr_test_1 = {
+    resource_group_ref = "rg_test"
+    name               = "acrtestdevne01"
+    sku                = "Premium"
+
+    georeplications = {
+      georeplication_test_1 = {
+        location                = "West Europe"
+        zone_redundancy_enabled = true
+        tags                    = { Owner = "prod" }
+      }
+      georeplication_test_2 = {
+        location                = "East US"
+        zone_redundancy_enabled = true
+      }
+    }
+
+    private_endpoint = {
+      name       = "test-endpoint"
+      subnet_ref = "vnet_test/snet_private_endpoint_1"
+      private_service_connection = {
+        name                 = "test-privateserviceconnection"
+        is_manual_connection = true
+      }
```

Comment:
If we use a private service connection with manual connection for most resources, it would make sense to set this configuration by default and only allow disabling or modifying it.
Check our current usage patterns.